### ISSUE TITLE
Prioritize single accessModeSufficient values

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2302]
As discussed on the mailing list starting with @madeleinerothberg's [comment](https://lists.w3.org/Archives/Public/public-epub-wg/2022May/0030.html):

> The thing about accessModeSufficient…. It’s the entries with single modes that are most useful to users, probably, because it is the users with restrictions on their perceptual modes who most need to know if they can use a particular resource. We can list out multiple sets of accessModeSufficient and there is no reason not to, but the most impactful entries will be those with single modes or the LACK of them.

Should we recommend the single values be listed and make other sets optional in the techniques document, or should we only emphasize the importance of single values?

**Comment by madeleinerothberg** [comment]
I will add that we don't want to encourage people to list single access modes if they are not sufficient! If people think they are supposed to simply take each accessMode entry and put it in accessModeSufficient that will be counterproductive.

**Comment by clapierre** [comment]
I think emphasizing single AMS entries and can give another combination for the most common other option and give an example of when these would be uses.  ie. a Children's Picture Book we might expect to see accessModeSufficient="visual".  
And when accessModeSufficient="textual" where there may be A) only text in the EPUB, vs. B) an EPUB with images and text where all images are described with a meaningful text description would be appropriate.
